src/publicLinkAccess.js

```diff
@@ -251,5 +251,23 @@ class PublicFiles {
       return Promise.reject(this.helpers.buildHttpErrorFromDavResponse(result.status, result.body))
     })
   }
+
+  /**
+   * Returns the file info for the given public resource
+   * @param   {string} token public share token - may contain the path as well
+   * @param   {string|null} password public link's password
+   * @param   {Object.<string, string>} properties WebDAV properties
```

Comment:
According to `list` docblock this should be of type `array` (`* @param {array}       properties`)

Reply:
Right - fixed.